src/test/kotlin/stringCalculator/EvaluationKtTest.kt

```diff
@@ -0,0 +1,37 @@
+package stringCalculator
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class EvaluationKtTest {
```

Comment:
```suggestion
internal class EvaluationTest {
```
클래스명+Test로 작성해주시는 것이 좋아요